JavaDoc warning when referencing Lombok generated methods

There is following JavaDoc warning during compilation
```
[WARNING] Javadoc Warnings
[WARNING] /.../powerauth-server/powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/CacheConfiguration.java:44: warning: reference not found: CallbackUrlEntity#getId()
[WARNING] * {@link CallbackUrlEntity#getId()} is used as a cache key.
[WARNING] ^
[WARNING] 1 warning
```
That is probably caused by missing Lombok generated methods during JavaDoc generation. Similar issue were already solved here https://github.com/wultra/powerauth-push-server/pull/867

--- a/pom.xml
+++ b/pom.xml
@@ -227,11 +227,39 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok-maven-plugin</artifactId>
+                <version>1.18.20.0</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.projectlombok</groupId>
+                        <artifactId>lombok</artifactId>
+                        <version>${lombok.version}</version>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <outputDirectory>${project.build.directory}/delombok</outputDirectory>
+                    <sourceDirectory>${project.basedir}/src/main/java</sourceDirectory>
+                    <addOutputDirectory>false</addOutputDirectory>
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>delombok</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>${maven-javadoc-plugin.version}</version>
                 <configuration>
+                    <sourcepath>${project.build.directory}/delombok</sourcepath>
                     <failOnError>false</failOnError>
                 </configuration>
                 <executions>

